### PR TITLE
[TDF] Misc fixes

### PR DESCRIPTION
**Pull request by bluehood**

### CODE REVIEW

**Comment by martinmine** [comment]
@phsft-bot build

**Comment by bluehood** [comment]
Cannot comply with clang-format because of markdown/doxygen requirements.

**Comment by martinmine** [comment]
@phsft-bot build

**Comment by martinmine** [comment]
@phsft-bot build

File: tree/treeplayer/inc/ROOT/TDFNodes.hxx [hunk 1]
```diff
@@ -85,7 +86,8 @@ public:
    TLoopManager(TTree *tree, const ColumnNames_t &defaultBranches);
    TLoopManager(ULong64_t nEmptyEntries);
    TLoopManager(const TLoopManager &) = delete;
-   ~TLoopManager(){};
+   TLoopManager &operator=(const TLoopManager &) = delete;
```
Comment (dpiparo) [inline tree/treeplayer/inc/ROOT/TDFNodes.hxx]: For consistency here shouldn't we have a default destructor? (very minor...)

**Comment by dpiparo** [comment]
I merge. The default destructor can be  addressed later.

**Pull request merged**